### PR TITLE
CLDR-16346 Followup to PR 2731

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-16346

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true
DISABLE_JIRA_ISSUE_MATCH=true

### CODE REVIEW

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java [hunk 1]
```diff
@@ -100,9 +100,8 @@ public class ShowLocaleCoverage {
             ;
 
     private static final String PROPERTIES_HEADER =
-        // Extra tabs are for github table formatting
         "# coverageLevels.txt\n"
-        + "# Copyright © 2022 Unicode, Inc.\n"
+        + "# Copyright © 2023 Unicode, Inc.\n"
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/tool/ShowLocaleCoverage.java:104]: should use `CldrUtililty.getCopyrightString("#")` - maybe next time

**Review by srl295 — APPROVED** [review]

**Comment by macchiati** [comment]
I can't check in because of a bad commit message, and the usual "Details" fix doesn't work:

`NOTICE:` Your previous squash attempt failed with code 422. If you get this error repeatedly, you may not have permission to push to this repository.`

**Comment by srl295** [comment]
> I can't check in because of a bad commit message, and the usual "Details" fix doesn't work:
> 
> 
> 
> `NOTICE:` Your previous squash attempt failed with code 422. If you get this error repeatedly, you may not have permission to push to this repository.`

FYI @sffc what is this ?

Ok.  You can disable the check. I'll dig out the info.

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
Hmm. It didn't disable the validation message. 

I get 404 trying to squash, can you try again?

**Comment by srl295** [comment]
> [X]  Maintainers are allowed to edit this pull request.
> _If checked, users with write access to unicode-org/cldr can add new commits to your … branch. You can always change this setting later._

It looks like I'm NOT allowed to force push to your branch even though you checked this.

**Comment by macchiati** [comment]
Still fails:

You are about to OVERWRITE the CLDR-16346-Followup-to-PR-#2731 branch in
macchiati/cldr <https://github.com/macchiati/cldr>.

 Yes, I understand. This is what I want.

NOTICE: Your previous squash attempt failed with code 422. If you get this
error repeatedly, you may not have permission to push to this repository.

**Comment by srl295** [comment]
@macchiati can you try it again now that i took the `#` out of the PR title

**Comment by srl295** [comment]
https://github.com/unicode-org/jira-github-pr-check/issues/28 referenced a 422 error when there was an emoji in the title

**Comment by srl295** [comment]
CLDR-16346 had the title `BRS v43 in Resolution+, Mark` 

I dropped the `+,` - @macchiati  can you try the squash again?

**Comment by macchiati** [comment]
That didn't help. sigh

**Comment by macchiati** [comment]
sounds good!

**Pull request closed without merging**